src/api-doc/api-doc.yaml

```diff
@@ -679,7 +680,7 @@ paths:
             type: string
         - in: query
           name: type
-          description: type we need to pass to match the entityType
+          description: entity type of the entities we need to pass to match the entityType
```

Comment:
@MallanagoudaB bad description :  
Please pass entity type value for the entity, 
check for similar changes